src/lib/Form/Type/Content/BaseContentType.php

```diff
@@ -74,9 +73,7 @@ public function configureOptions(OptionsResolver $resolver)
                     return $value;
                 }
 
-                return $options['userUpdateStruct']
-                    ?? $options['userCreateStruct']
-                    ?? $options['contentUpdateStruct']
+                return $options['contentUpdateStruct']
```

Comment:
Why are we removing these?

Reply:
Because those options never existed and were added here by mistake - even more, when you think that BaseContentType now should now about UserType as well.